### PR TITLE
feat: Refresh Interval Options

### PR DESCRIPTION
**Pull request by coreywillwhat**
## Overview

This update introduces several enhancements and fixes aimed at improving the integration's configuration flexibility and efficiency in polling mechanisms.

### Enhancements
- **Configurable Scan Interval**: Integration of `OptionsFlowHandler` allows users to adjust the scan interval via the `Configure` menu, enhancing user control.
- **Efficient Polling**: Adoption of `async_track_time_interval` for polling initiation at startup removes the need for manual scheduling in `AudiAccount.update`, streamlining the update process.
- **Localization Updates**: Made updates to strings and translations to ensure clarity and accessibility.

### Technical Notes
- **Restart Requirement**: Changes necessitate a system restart to take effect. This requirement is detailed in the `data_description` within translations, ensuring users are informed via the UI.
- **Polling Mechanism Overhaul**: Leveraging `async_track_time_interval` from the get-go allows for interval-based polling using Home Assistant's built-in scheduling tools, bypassing the need for hardcoded scheduling.
- **Future-Proofing `AudiAccount.update`**: With these changes, `AudiAccount.update` is now freed up for potential use in front-end service calls, with plans for a follow-up PR pending this update's approval and merge.
- **Effort Towards User-Controlled Polling**: Next steps involve utilizing the `coordinator` for more flexible update scheduling, aiming to give users the ability to toggle polling on and off directly from the UI.

### CODE REVIEW

**Review by cdnninja — APPROVED** [review]

**Comment by cdnninja** [comment]
You are rocking in!  Thanks for all the effort to improve this integration!

**Comment by coreywillwhat** [comment]
Do we want to wait for 2 approvals or just 1 for these to merge? What are we thinking? I saw you started this conversation over in the discussions but not sure there was a conclusion.

**Comment by Kolbi** [comment]
> Do we want to wait for 2 approvals or just 1 for these to merge? What are we thinking? I saw you started this conversation over in the discussions but not sure there was a conclusion.

Good question, but if cdnninja checked code for me it's fine, I also had a quick view above the code changes and merged it.

**Pull request merged**
